apps/services/auth/ids-api/src/app/delegations/test/delegations-filters-test-cases.ts

```diff
@@ -0,0 +1,259 @@
+import {
+  clientId,
+  customScopes,
+  legalGuardianScopes,
+  procurationHolderScopes,
+  representativeScopes,
+  TestCase,
+} from './delegations-filters-types'
+import { createNationalId } from '@island.is/testing/fixtures'
+import { createClient } from '@island.is/services/auth/testing'
+
+const person1 = createNationalId('person')
+const person2 = createNationalId('person')
+const company1 = createNationalId('company')
+const company2 = createNationalId('company')
+
+export const testCases: Record<string, TestCase> = {
+  // Returns available delegations for legal guardians
```

Comment:
What about returning and/or filtering the authenticated user? Is that implemented? Would be nice to see tests for that.

Reply:
The authenticated user is handled in the IDS only for now.